rotkehlchen/tests/exchanges/test_binance.py

```diff
@@ -944,17 +944,22 @@ def mock_my_lendings(url, *args, **kwargs):  # pylint: disable=unused-argument
 
     with (
         patch.object(binance.session, 'request', side_effect=mock_my_lendings),
-        binance.db.conn.cursor() as cursor,
+        binance.db.conn.read_ctx() as cursor,
     ):
-        assert binance.query_lending_interests_history(
-            cursor=cursor,
-            start_ts=Timestamp(0),
-            end_ts=Timestamp(API_TIME_INTERVAL_CONSTRAINT_TS),
-        ) is False
-
-        assert cursor.execute(
-            'SELECT COUNT(*) FROM history_events WHERE subtype="reward";',
-        ).fetchone()[0] == 4
+        for count, location in [
+            (0, Location.BINANCEUS),
+            (4, Location.BINANCE),
+        ]:
+            binance.location = location
+            assert binance.query_lending_interests_history(
+                cursor=binance.db.conn.cursor(),
+                start_ts=Timestamp(0),
+                end_ts=Timestamp(API_TIME_INTERVAL_CONSTRAINT_TS),
+            ) is False
+
+            assert cursor.execute(
+                'SELECT COUNT(*) FROM history_events WHERE subtype="reward";',
+            ).fetchone()[0] == count
```

Comment:
this should have a location filter. If you change the order in the list I guess this one would fail